src/test/java/nl/atos/ontwikkelstraat/FormControllerSteps.java

```diff
@@ -0,0 +1,146 @@
+import cucumber.api.java.en.When;
+import nl.atos.ontwikkelstraat.pojo.FormData;
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+
+public class FormControllerSteps {
+    private final WebDriver webdriver;
+    public FormData formData;
+
+    public FormControllerSteps(SharedDriver webdriver) {
+        this.webdriver = webdriver;
+    }
+
+    @Before
+    public void setUp() {
+        formData = new FormData("john", "banks", "frontstreet", 12, "9485KS");
+    }
+
+    @After
+    public void tearDown() {
+        formData = null;
+    }
+
+    @Given("^That the website is running and I navigated to it$")
+    public void that_the_website_is_running_and_I_navigated_to_it() throws InterruptedException {
+        webdriver.manage().window().maximize();
+        webdriver.manage().timeouts().setScriptTimeout(10, TimeUnit.SECONDS);
+        if (System.getenv("TRAVIS") == "true") {
+            webdriver.get("localhost:4445");
+        } else webdriver.get("http://localhost:8080/");
+
+    }
+
+    @When("^I fill in my firstName with \"([^\"]*)\"$")
+    public void i_fill_in_my_firstName_with(String firstname) {
+        formData.setName(firstname);
+        webdriver.findElement(By.id("name")).sendKeys(firstname);
+        assertEquals(firstname, webdriver.findElement(By.id("name")).getAttribute("value"));
+        //System.out.println(webdriver.findElement(By.id("name")).getAttribute("value"));
+    }
+
+
+    @And("^I fill in my lastName$")
+    public void i_fill_in_my_lastName() throws Throwable {
+        formData.setSurname("Johnson");
+        webdriver.findElement(By.id("surname")).sendKeys("Johnson");
+        assertEquals("Johnson", webdriver.findElement(By.id("surname")).getAttribute("value"));
+    }
+
+    @And("^I fill in my StreetName$")
+    public void i_fill_in_my_StreetName() throws Throwable {
+        formData.setStreetName("hoofdstraat");
+        webdriver.findElement(By.id("streetName")).sendKeys("hoofdstraat");
+        assertEquals("hoofdstraat", webdriver.findElement(By.id("streetName")).getAttribute("value"));
+    }
+
+    @And("^I fill in my HouseNumber$")
+    public void i_fill_in_my_HouseNumber() throws Throwable {
+        formData.setHouseNumber(134);
+        webdriver.findElement(By.id("houseNumber")).sendKeys("134");
+        assertEquals("134", webdriver.findElement(By.id("houseNumber")).getAttribute("value"));
+    }
+
+    @And("^I fill in my Zipcode$")
+    public void i_fill_in_my_Zipcode() throws Throwable {
+        formData.setZipCode("9485SL");
+        webdriver.findElement(By.id("zipCode")).sendKeys("9485SL");
+        assertEquals("9485SL", webdriver.findElement(By.id("zipCode")).getAttribute("value"));
+    }
+
+    @And("^I hit the submit button$")
+    public void i_hit_the_submit_button() throws Throwable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/vinsterdamus/Ontwikkelstraat/file/2345692422/issues/source?bid=3138095&fileBranchId=3208141#l85)